pkg/makefile

```diff
@@ -197,7 +197,7 @@ rpm: stage_rpm
 		-d device-mapper-event \
 		-d device-mapper-event-libs \
 		-d device-mapper-libs \
-		-d 'zenoss-docker = 1.7.1' \
+		-d 'docker-engine = 1.7.1' \
```

Comment:
This should be thoroughly tested by a developer RE upgrades before being merged.  Also, this will require customers to add Docker's yum repo to their system - that process will need to be automated somehow.